src/reference/config/writing-platform-configs.rst

```diff
@@ -49,277 +49,276 @@ Why Were Platforms Introduced?
+on that platform.
 
-As a result the above configuration can be simplified to:
+Cylc will attempt to communicate with jobs via the other login node if either
+of the login_nodes becomes unavailable.
 
-.. code-block:: cylc
-   :caption: part of a ``global.cylc`` config file
+With multiple hosts defined under ``slurm_cluster``, a job runner is required.
 
-   [platforms]
-       [[pbs_cluster]]
-           job runner = pbs
+.. note::
 
+   The "background" and "at" job runners require single-host platforms,
+   because the job ID is only valid on the submission host.
+
+We have set ``retrieve job logs = True``. This will ensure our job logs are
+fetched from the ``slurm_cluster`` platform. This setting is recommended for
+any remote platform (i.e. where install target is not localhost).
 
 
-Two Similar Clusters
-^^^^^^^^^^^^^^^^^^^^
+Grouping Platforms
+^^^^^^^^^^^^^^^^^^
 
 - **Platform groups allow users to ask for jobs to be run on any
   suitable computer.**
 
 .. admonition:: Scenario
 
-   Your site has two mirrored clusters with separate PBS queues and
-   file systems. Users don't mind which cluster is used and just
-   want to set ``flow.cylc[runtime][mytask]platform = supercomputer``:
-
-   Remember, because the install target defaults to the platform name
-   clusterA and clusterB have different install targets.
-
-.. spelling:word-list::
-
-   clusterA
-   clusterB
+   Extending the example from above, we now wish to set the ``slurm_cluster``
+   up such that one host from ``slurm_cluster`` can accept background jobs.
+   We would like to group these platforms together so users can set
+   :cylc:conf:`flow.cylc[runtime][<namespace>]platform` = supercomputer.
 
 .. code-block:: cylc
    :caption: part of a ``global.cylc`` config file
 
    [platforms]
-       [[clusterA]]
-           hosts = login_node_A1, login_node_A2
-           batch system = pbs
-       [[clusterB]]
-           hosts = login_node_B1, login_node_B2
-           batch system = pbs
+       [[slurm_cluster, slurm_cluster_bg]]  # settings that apply to both:
+           install target = slurm_cluster
+           retrieve job logs = True
+       [[slurm_cluster]]
+           batch system = slurm
+           hosts = login_node_1, login_node_2
+       [[slurm_cluster_bg]]
+           hosts = login_node_1
        [platform groups]
            [[supercomputer]]
-           platforms = clusterA, clusterB
-
-.. note::
-
-   Why not just have one platform with all 4 login nodes?
-
-   Having hosts in a platform means that Cylc can communicate with
-   jobs via any host at any time. Platform groups allow Cylc to
-   pick a platform when the job is started, but Cylc will not then
-   be able to communicate with that job via hosts on another
-   platform in the group.
-
-
-Preferred and Backup Hosts and Platforms
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+               platforms = slurm_cluster, slurm_cluster_bg
```

Comment:
I don't think it makes sense to have a group mixing slurm & background jobs.
The example I was thinking of is wanting to run a background job on either of the login nodes so the config would be:
```
[platforms]
    [[slurm_cluster, slurm_cluster_bg1, slurm_cluster_bg2]]  # settings that apply to all:
        install target = slurm_cluster
        retrieve job logs = True
    [[slurm_cluster]]
        hosts = login_node_1, login_node_2
        job runner = slurm
    [[slurm_cluster_bg1]]
        hosts = login_node_1
    [[slurm_cluster_bg2]]
        hosts = login_node_2
[platform groups]
    [[slurm_cluster_bg]]
        platforms = slurm_cluster_bg1, slurm_cluster_bg2
```